database/src/migrations/release.0.16/biohub.sql

```diff
@@ -675,17 +593,19 @@ COMMENT ON TABLE management_action_type IS 'List of Management Actions.'
 ;
 
 -- 
--- TABLE: no_sample_permit 
+-- TABLE: permit 
 --
 
-CREATE TABLE no_sample_permit(
+CREATE TABLE permit(
     id                           integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    p_id                         integer,
+    s_id                         integer,
```

Comment:
I wonder if it would be better to handle survey permits like you did with survey funding, where its a separate table with a project permit id foreign key.

Mostly thinking about the case where a single project permit is assigned to multiple surveys.  Unless this never happens?

Reply:
1) a permit can be associated with one or zero projects
2) a permit can only be associated with one survey
3) a permits association with a survey implies that sampling was conducted

the model currently supports the above.